### PR TITLE
[PAYSHIP-3121] Apple Pay on PS 1.7

### PR DESCRIPTION
**Pull request by L3RAZ**

### CODE REVIEW

**Review by CorentinFlament — DISMISSED** [review]

**Review by Matt75 — DISMISSED** [review]

File: ps_checkout.php [hunk 1]
```diff
@@ -105,16 +106,19 @@ class Ps_checkout extends PaymentModule
...
+        'PS_CHECKOUT_DOMAIN_REGISTERED_LIVE' => false,
     ];
 
     public $confirmUninstall;
     public $bootstrap;
 
     // Needed in order to retrieve the module version easier (in api call headers) than instanciate
     // the module each time to get the version
-    const VERSION = '7.3.6.3';
+    const VERSION = '7.4.2.2';
```
Comment (seiwan) [inline ps_checkout.php:119]: It can be 7.4.2.0, actually, as you prefer.

File: ps_checkout.php [hunk 2]
```diff
@@ -133,7 +137,7 @@ public function __construct()
 
         // We cannot use the const VERSION because the const is not computed by addons marketplace
         // when the zip is uploaded
-        $this->version = '7.3.6.3';
+        $this->version = '7.4.2.2';
```
Comment (seiwan) [inline ps_checkout.php:140]: It can be 7.4.2.0, actually, as you prefer.

File: config.xml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <module>
     <name>ps_checkout</name>
     <displayName><![CDATA[PrestaShop Checkout]]></displayName>
-    <version><![CDATA[7.3.6.3]]></version>
+    <version><![CDATA[7.4.2.2]]></version>
```
[the remark this answers is not in the document]
Comment (L3RAZ) [reply config.xml:5]: I will keep 4.2.2 Just so we know that capture retries and other fixes are added

File: ps_checkout.php (hunk 1 above)
Comment (L3RAZ) [reply ps_checkout.php:119]: Same here

File: ps_checkout.php (hunk 2 above)
Comment (L3RAZ) [reply ps_checkout.php:140]: And here

**Review by seiwan — DISMISSED** [review]

**Comment by seiwan** [comment]
Approved, but there is some issues on the CI/CD

File: .github/workflows/checkout-cd-prod-beta.yml
Comment (seiwan) [inline .github/workflows/checkout-cd-prod-beta.yml:1]: Actually, I think we can delete this workflow.

**Review by seiwan — CHANGES_REQUESTED** [review]

**Review by seiwan — APPROVED** [review]

**Pull request merged**